src/e2e-test/features/dataplex/source/DesignTimeWithMacro.feature

```diff
@@ -0,0 +1,67 @@
+@Dataplex_Source
+Feature: Dataplex Source - Verification of pipeline preview and deploy with macros
+
+  @BATCH-TS-DATAPLEX-MACRO-01
```

Comment:
Combine @BATCH-TS-DATAPLEX-MACRO-01 and @BATCH-TS-DATAPLEX-MACRO-02 in single scenario.

No need of separate scenarios.
You can run the preview and then deploy it and run in single scenario.
Also steps for capturing logs are missing